dd-sdk-android/src/main/kotlin/com/datadog/android/Datadog.kt

```diff
@@ -283,6 +284,19 @@ object Datadog {
         }
     }
 
+    /**
+     * For Datadog internal use only.
+     *
+     * @see _InternalProxy
+     */
+    @JvmStatic
```

Comment:
I don't think we need this annotation here, because this property is supposed to be called only by us, so no interop with Java is needed (both Flutter and RN use Kotlin for the native modules).